mod/item.php

```diff
@@ -640,7 +641,7 @@ function item_post(&$a) {
 			if (dbm::is_result($r)) {
 				if(strlen($attachments))
```

Comment:
Standards: Could you please add brackets to this conditional statement? 😊
Standards: Could you please add a space after the `if`? 😊